prog/encoding.go

```diff
@@ -83,6 +83,8 @@ func (ctx *serializer) call(c *Call) {
 
 	anyChangedProps := false
 	c.Props.ForeachProp(func(name, key string, value reflect.Value) {
+		// reflect.Value.IsZero is added in go1.13, not available in Appengine SDK.
+		// if reflect.DeepEqual(value.Interface(), reflect.Zero(value.Type()).Interface()) {
```

Comment:
If everything goes as expected, CI will detect an issue here. Then I will fix this commit.